code/mob/living/life/chems.dm

```diff
@@ -10,8 +10,12 @@
 		if (owner.reagents)
 			var/reagent_time_multiplier = get_multiplier()
 
+
 			owner.reagents.temperature_reagents(owner.bodytemperature, 100, 35/reagent_time_multiplier, 15*reagent_time_multiplier)
 
+			if(owner && HAS_MOB_PROPERTY(owner, PROP_TEMP_CHEM_DEPLETION))
+				reagent_time_multiplier *= get_temperature_mult()
```

Comment:
this is affecting more than just depletion - this affects the lag-compensation multiplier passed for depletion *and* effects of chems

Reply:
While I meant for that overall effect, yeah it'd make a lot more sense to have the properties do the things they say they do.